CMakeLists.txt

```
@@ -155,10 +161,14 @@ if(SFML_USE_SYSTEM_DEPS)
     list(REMOVE_DUPLICATES CMAKE_IGNORE_PATH)
 endif()
 
-# Visual C++: remove warnings regarding SL security and algorithms on pointers
 if(SFML_COMPILER_MSVC)
     # add an option to choose whether PDB debug symbols should be generated (defaults to true when possible)
     sfml_set_option(SFML_GENERATE_PDB TRUE BOOL "True to generate PDB debug symbols, FALSE otherwise.")
+
+    # if building using a compiler launcher, embed the MSVC debugging information to allow for caching
+    if(CMAKE_C_COMPILER_LAUNCHER OR CMAKE_CXX_COMPILER_LAUNCHER)
```

```suggestion
    if(CMAKE_CXX_COMPILER_LAUNCHER)
```
As of #2767 we can forget that C compilers exist.